5.py

```diff
@@ -0,0 +1,79 @@
+import random
```

Comment:
This solution is overcomplicated. Sometimes it is better to write simple things according to the tasks. But I'm impressed you already know a lot about how to write Python code